spectator/meter/monotonic_counter.go

```diff
@@ -0,0 +1,38 @@
+package meter
+
+import (
+	"fmt"
+	"github.com/Netflix/spectator-go/spectator/writer"
+)
+
+// MonotonicCounter is used to measure the rate at which some event is occurring. This
+// type is safe for concurrent use.
+//
+// The value is a monotonically increasing number. A minimum of two samples must be received
+// in order for spectatord to calculate a delta value and report it to the backend.
+//
+// A variety of networking metrics may be reported monotonically and this metric type provides a
+// convenient means of recording these values, at the expense of a slower time-to-first metric.
```

Comment:
There are no atlas-docs on this meter type, so I just inlined the spectatord description.